cmake.deps/CMakeLists.txt

```diff
@@ -142,6 +142,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
 endif()
 
 include(ExternalProject)
+set_directory_properties(PROPERTIES EP_PREFIX "${DEPS_BUILD_DIR}")
```

Comment:
I can revert this if people think defining `PREFIX` for each dependency is more explicit/clearer.